clients/web/test/testUtils.js

```diff
@@ -1219,6 +1225,25 @@ $(function () {
 girderTest.startApp = function () {
     var defer = new $.Deferred();
     girderTest.promise.then(function () {
+        /* Track bootstrap transitions using our own test event. */
+        $.support.transition = {end: 'girdertest_transitionend'};
```

Comment:
It's possible I'm not understanding this correctly, but looking at what bootstrap does with this, I think it is expecting this to be a browser specific event name.  I don't think you should be changing it.